src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/IntegrationTestBase.kt

```diff
@@ -446,6 +454,36 @@ abstract class IntegrationTestBase {
       .returnResult(ActionPlanReviewsResponse::class.java)
       .responseBody.blockFirst()!!
 
+  fun getInductionSchedule(prisonNumber: String): InductionScheduleResponse =
```

Comment:
Added these methods to get the induction schedule, and induction schedule history, via the REST APIs